build.sh

```diff
@@ -37,7 +37,7 @@ HELP="$0 [clean] [libcudf] [cudf] [dask_cudf] [benchmarks] [tests] [libcudf_kafk
    --disable_nvtx       - disable inserting NVTX profiling ranges
```

Comment:
Need to add 2021 copyrights on this file

Reply:
ack, would be great to add stuff like this to a pre-commit hook